app/models/publication.rb

```diff
@@ -164,7 +166,7 @@ def pubhash_needs_update!(*_args)
   end
 
   def pubhash_needs_update?
-    @pubhash_needs_update || false
+    pubhash_needs_update || false
```

Comment:
aren't there other uses of @pubhash_needs_update in the publication model?

Reply:
Really just the `pubhash_needs_update!` setter, which I opted to let assign the variable directly.